charts/baseChart.js

```diff
@@ -28,9 +29,26 @@ module.exports = class Chart extends React.Component {
             );
         }
         if (this.state.data) {
+            const config = {
+                title: {
+                    text: this.props.config.graphTitle
+                },
+                yAxis: {
+                    title: {
+                        text: this.props.config.graphYAxis
+                    }
+                },
+                xAxis: {
+                    title: {
+                        text: this.props.config.graphXAxis
+                    }
+                },
+                series: this.state.data
```

Comment:
What do you think about moving these values off of config and onto props instead? props is essentially a config already.